### ISSUE TITLE
Undetected violation of comment indentation after a distributed expression

### ISSUE THREAD

**Issue by SGanguly1999** [checkstyle/checkstyle#9945]
I have read check documentation: https://checkstyle.org/config_misc.html#CommentsIndentation
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
/var/tmp $ javac Hello.java


/var/tmp $ cat config.xml

<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
<module name = "Checker">
   <module name="TreeWalker">
      <module name="CommentsIndentation"/>
   </module>
</module>


/var/tmp $ cat Hello.java

class Hello {
    void foo55() {
            /* violation */
        new Object()
            .toString();
            // comment
    }
}

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.42-all.jar -c config.xml Hello.java

Starting audit...
[ERROR] /var/tmp/Hello.java:3:13: Block comment has incorrect indentation level 12, expected is 8, indentation should be the same level as line 4. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.

```

---

**Expected**

```
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.42-all.jar -c config.xml Hello.java

Starting audit...
[ERROR] /var/tmp/Hello.java:3:13: Block comment has incorrect indentation level 12, expected is 8, indentation should be the same level as line 4. [CommentsIndentation].
[ERROR] /var/tmp/Hello.java:6:13: Comment has incorrect indentation level 12, expected is 8, indentation should be the same level as line 4. [CommentsIndentation]
Audit done.
Checkstyle ends with 2 errors.
```

Checkstyle should report violation on  line 6 of the code.

**Comment by SGanguly1999** [comment]
I am on it.
